packages/babel-traverse/src/path/replacement.ts

```diff
@@ -40,7 +40,7 @@ import hoistVariables from "@babel/helper-hoist-variables";
 
 export function replaceWithMultiple(
   this: NodePath,
-  nodes: t.Node[],
+  nodes: t.Node | t.Node[],
```

Comment:
`this._verifyNodeList` will cast `Node` to `Node[]`.

Reply:
We seem to have no need to declare support, the recommended usage is still `replaceWithMultiple([node])`;

Reply:
The signature `replaceWithMultiple(node: t.Node)` is used in https://github.com/babel/babel/blob/d85bf6fa12d8bbaa468f634c129520f8ffe1c38b/packages/babel-plugin-transform-for-of/src/no-helper-implementation.ts#L32-L37